pkg/conversion/queryparams/convert.go

```diff
@@ -92,11 +107,11 @@ func Convert(obj runtime.Object) (url.Values, error) {
 	case reflect.Ptr, reflect.Interface:
```

Comment:
Not your fault, but is the case of struct-wrapped-in-interface tested? From looking at this, I would not expect it to work.